### PR TITLE
cargo-edit 0.13.0

### PR DESCRIPTION
**Pull request by CosmicHorrorDev**
Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>## 0.13.0 - 2024-09-16

### Breaking Changes

- *(upgrade)* Remove of `--offline` as we don't have local caching for sparse registry

### Fixes

- *(upgrade)* Switched from git to sparse registry for faster runs</pre>
</details>

### CODE REVIEW

File: Formula/c/cargo-edit.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class CargoEdit < Formula
   homepage "https://killercup.github.io/cargo-edit/"
   # TODO: check if we can use unversioned `libgit2` at version bump.
   # See comments below for details.
```
Comment (CosmicHorrorDev) [inline Formula/c/cargo-edit.rb:5]: I would assume the CI failure is probably related to this, but unfortunately I don't know enough about brew to really help :sweat_smile:

Comment (carlocab) [reply Formula/c/cargo-edit.rb]: The comments below describe exactly what you needed to do in detail. I've done that for you now.

Comment (CosmicHorrorDev) [reply Formula/c/cargo-edit.rb]: Was slowly working through that myself, but glad that you handled it for me. Thanks :heart:

Comment (carlocab) [reply Formula/c/cargo-edit.rb]: Huh, looks like it'll take bit more than what was in the comments. I'll have a look again tomorrow.

**Review by chenrui333 — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Review by iMichka — APPROVED** [review]

**Pull request merged**